src/firefox/preferences.js

```diff
@@ -65,13 +69,13 @@ prefs.common = {
 };
 
 // Prefs specific to Firefox for Android.
-prefs.fennec = {
+const prefsFennec:FirefoxPreferences = {
```

Comment:
@pdehaan it seems a great idea, especially now that we are expanding the flow type definitions, I'll try it asap.